.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -38,24 +38,21 @@ Test case added to lib/integration.sh
 If no, create an issue with enhancement/testing label
 -->
 
-**Will this break upgrades and downgrades. Has it been tested?**:
+**Will this break upgrades or downgrades. Has updating a running cluster been tested?**:
 
 
-**Does this require config only upgrades?**:
+**Does this require configuration changes to work?**:
```

Comment:
Can we make this bit more clear? What configuration changes are these?

Reply:
Is this more clear?

**Does this change require updates to the CNI daemonset config files to work?**: